server/src-lib/Hasura/Server/Query.hs

```diff
@@ -135,20 +135,19 @@ fetchLastUpdate = do
        ORDER BY occurred_at DESC LIMIT 1
           |] () True
   case l of
```

Comment:
Could this use the `FromRes (Maybe a)` instance with `withQE` to avoid needing to do the explicit pattern matching?